README.md

```diff
@@ -67,6 +67,28 @@ Baseline | Base/Idle cluster | Captures performance data of the base cluster wit
 
 NOTE: Services per namespace cluster limits test is covered by Deployments per ns test, creating a separate job for it is in progress.
 
+
+### Modifying/Adding new workloads to the scale-ci-pipeline
+
+Adding new workloads is managed by scale-ci-watcher and it expects the following:
+- JJB template: It is the job definition in yaml ( xml is also supported, watcher takes care of converting it to yaml )
+- Pipeline-build script for the Job and a properties file: The script is responsible for building the job as a stage looking with the configutation/parameters specified in the properties file.
+
+There are already a bunch of jobs integrated into the scale-ci-pipeline, so we can resuse most of the stuff and just plugin the job specific stuff. Let's take the example of nodevertical scale test, here are the instructions:
```

Comment:
s/resuse/reuse

My suggested write of this:

There are many existing jobs in scale-ci-pipeline which can serve as an example/template job. One example is the NodeVertical scale test.  To create a new workload using NodeVertical as a template, follow these steps:

Reply:
Sounds good, fixed it.